### ISSUE TITLE
Bash 4 default conf file syntax error operand expected

### ISSUE THREAD

**Issue by modlin** [coderofsalvation/kanban.bash#15]
Preconditions:

* GNU bash, version 4.4.12
* kanban.bash as of 90d0a93 2018-09-21
* Debian GNU/Linux 9.6

Steps:

1. Be sure you have no existing `~/.kanban.conf` file before doing this test to see the error
2. Get the latest `kanban` script ( 90d0a93 2018-09-21 ) for example following the official install instructions at https://github.com/coderofsalvation/kanban.bash
3. `./kanban`

Result:

    /home/you/.kanban.conf: line 8: 'HOLD': syntax error: operand expected (error token is "'HOLD'")
    Usage:
    ...

Expected:

    Usage:
    ...

* in other words show the usage, without showing that error

Related:

* Issue #5 Error with the default conf file, but srkiNZ84 `fixed by changing the configuration file to ... a lower case "-a"` does not work for [bash 4](http://www.tldp.org/LDP/abs/html/bashver4.html)

**Comment by seflue** [comment]
I also stumbled over this issue. I fixed it changing the quotes around the indices declaring the WIP limits in the config from single quote to double quote, which also works.

```bash
# ... previous stuff ... #
config_example="# kanban config file

# kanban config file
statuses=('BACKLOG' 'HOLD' 'DOING' 'CODE' 'DONE') 

# maximum amount of todos within status (triggers warning when exceeds)
declare -a maximum_todo
maximum_todo[\"HOLD\"]=5
maximum_todo[\"DOING\"]=4
maximum_todo[\"CODE\"]=3
"
# ... following stuff ... #
```

Note that the double quotes must be escaped in `kanban` executable, in the resulting `kanban.conf` they are unescaped.
